javascript/javascript1/week2/week-event.js

```diff
@@ -0,0 +1,9 @@
+function getEventWeekday(noOfEventDays)
+{
+    const today = new Date();
+    const futureEventDay= (today.getDay() + noOfEventDays) % 7;
+    console.log(futureEventDay);
+    const WeekDays=['Sunday','Monday','Tuesday','Wednesday','Thursday','Friday','Saturday'];
+    console.log(`${WeekDays[futureEventDay]}`);
```

Comment:
Use `return` and leave the console logging to whoever calls the function, [see homework](https://github.com/HackYourFuture-CPH/JavaScript/blob/master/javascript1/week2/homework.md#event-application):

```js
// With todays weekday a tuesday
console.log(getEventWeekday(9)); // Logs out "Thursday"
```